### PR TITLE
Babel

### PR DESCRIPTION
**Pull request by gcoombe**
Transpile via babel to support node 6 and up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +109,6 @@ Get an encrypted value to store in your configuration files.
 
 ## Dependencies
 
-We use async/await so node 8 and up is required.
+Node 6 and up is supported
```
Comment (dhessler) [inline README.md:112]: Why stop at node v6 here? The transpiled build supports node v4 (tested locally).

File: package.json [hunk 2]
```diff
@@ -20,7 +22,7 @@
   },
   "homepage": "https://github.com/fetch-auto/node-config-cryptex#readme",
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=6.0.0"
```
Comment (dhessler) [inline package.json:25]: tested and working with 4.0.0

File: README.md (hunk 1 above)
Comment (gcoombe) [reply README.md:112]: Mostly to reduce maintenance overhead moving forward. You are right that it should be fine currently but I would like to develop some additional features in this library moving forward and retaining backwards compatibility can be a bit of a pain.

Additionally, from a more philosophical perspective I don't want to be supporting an end of life version that won't be receiving security updates etc (which will be the case at the end of April).

Hope that makes sense!

**Review by randytarampi — APPROVED** [review]
LGTM.

File: README.md (hunk 1 above)
Comment (dhessler) [reply README.md:112]: makes perfect sense, thanks for the updates!

**Pull request merged**